src/components/Toast/theme.ts

```diff
@@ -0,0 +1,39 @@
+import { Themeable } from '@emotion/react';
+import { ToastParts, ToastProps } from 'src/components/Toast';
+
+export const toastTheme: Themeable<ToastProps, ToastParts> = {
+  baseStyle: {
```

Comment:
Everything work here as expected?

Reply:
Yes! it took a little bit longer than expected to get it set up, but I see why it's very beneficial.